package.json

```
@@ -12,7 +12,7 @@
     "lint": "tslint --fix -c tslint.json -t stylish -p ./tsconfig.json",
     "update": "npx npm-check -u",
     "prepublishOnly": "npm run build",
-    "pretest": "npx fastify-secure-session > secret-key",
+    "pretest": "npx @fastify/secure-session > secret-key",
```

this should work in win

```suggestion
    "pretest": "npx \"@fastify/secure-session\" > secret-key",
```